jib-maven-plugin/README.md

```diff
@@ -311,6 +320,43 @@ Configure credential helpers to use by specifying them as a `credHelper` for the
 </configuration>
 ```
 
+#### Using Specific Credentials
+
+You can specify credentials directly in the <auth> parameter for the `from` and/or `to` images. In the example below, `to` credentials are retrieved from the `REGISTRY_USERNAME` and `REGISTRY_PASSWORD` environment variables.
```

Comment:
Place `<auth>` in the black quotes. (Otherwise not visible at all.)

Reply:
Perhaps worth mentioning to use this as a last resort, as the plain password is entered into the file? (The command line apporach is not so different, as people on the same system can see them too.)